orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/rna/NovelSpliceJunction.java

```diff
@@ -0,0 +1,38 @@
+package com.hartwig.hmftools.datamodel.rna;
+
+import org.immutables.gson.Gson;
+import org.immutables.value.Value;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Gson.TypeAdapters
+@Value.Immutable
+@Value.Style(passAnnotations = { NotNull.class, Nullable.class })
+public interface NovelSpliceJunction {
```

Comment:
Add enter? 

Also, consider renaming package to isofox? Seems more consistent with other packages.